tools/add_triage_signature.py

```diff
@@ -612,6 +612,47 @@ def _update_ticket(self, url, issue_key, should_update=False):
             self._update_triaging_ticket(issue_key, report, should_update=should_update)
 
 
+class FeatureUsageSignature(Signature):
+    """
+    The signature adds information about features configured in cluster based
+    on feature_usage field. For example: OLM operators, NTP additional
+    source or proxy configuration
+    """
+    OLM_OPERATORS = ["cnv", "lso", "ocs"]
+
+    def __init__(self, jira_client):
+        super().__init__(jira_client, comment_identifying_string="h1. Configured Features")
+
+    @classmethod
+    def isOlm(cls, name):
+        return name.lower() in cls.OLM_OPERATORS
+
+    @classmethod
+    def isGeneralFeature(cls, name):
+        return not cls.isOlm(name)
+
+    def _update_ticket(self, url, issue_key, should_update=False):
+        url = self._logs_url_to_api(url)
+
+        metadata = get_metadata_json(url)
+        cluster = metadata['cluster']
+        feature_usage_field = cluster.get('feature_usage')
```

Comment:
Maybe a small comment saying `# A list of features, some of which are operators` so all the code after makes more sense to those unfamiliar with this field